src/views/dmca/dmca.scss

```diff
@@ -0,0 +1,2 @@
+@import "../../colors";
+@import "../../typography";
```

Comment:
@TheGrits this is good for now, thanks! We might do a bit of restructuring with the typography file, but this keeps to current convention.